<!DOCTYPE html>
<html id="protocol">

<head>
  <!-- Put fonts and css links here -->

  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">

  <title>Protocols</title>
  <meta content="" name="description">
  <meta content="" name="keywords">

  <!-- Favicons -->
  <link rel="icon" type="image/png" href="../../content/induceR-favicon-w.png">
  
  <!-- (Google) Fonts -->
  <!-- Main CSS-->
  <link href="../../styles/index.css" rel="stylesheet">

  <script type="text/javascript" src="../../script/functions.js"></script>

</head>

<body id="protocol-body">
  <!-- Navigation menu -->
  <div class="navbar">
    <img id="nav-logo" src="..\..\content\induceR-favicon-w.png" alt="induceR">
    <div class="dropdown">
      <button class="dropbtn">Documentation
        <i class="fa fa-caret-down"></i>
      </button>
      <div class="dropdown-content">
        <a href="protocols.html">Protocols</a>
        <a href="labnotebook.html">Lab Notebook</a>
        <a href="biosafety.html">Biosafety and Ethics</a>
        <a href="resources.html">Resources</a>
      </div>
    </div>

    <div class="dropdown">
      <button class="dropbtn">Project
        <i class="fa fa-caret-down"></i>
      </button>
      <div class="dropdown-content">
        <a href="../project/background.html">Background</a>
        <a href="../project/design.html">Design</a>
        <a href="../project/drylabs.html">Dry Labs</a>
        <a href="../project/wetlabs.html">Wet Labs</a>
        <a href="../project/results.html">Results</a>
      </div>
    </div>

    <div class="dropdown">
      <button class="dropbtn">About
        <i class="fa fa-caret-down"></i>
      </button>
      <div class="dropdown-content">
        <a href="../about/team.html">Our Team</a>
        <a href="../about/collab.html">Collaborators</a>
        <a href="../about/sponsors.html">Sponsors</a>
      </div>
    </div>

    <a href="../../index.html">Home</a>

  </div>
  <div class="banner">
    <img id="protocol-bg" src="../../content/banners/protocol_bg.svg" alt="protocol_bg">
    <h1 class="banner-head">Protocols</h1>
  </div>

  <img id="el-membrane" src="..\..\content\el_membrane.svg" alt="membrane">

  <div class="main">

    <div class="section1">
      <!-- Tabs -->
      <div class="tab-container">
        <div class="tab">
          <h2 class="tab-section">Our protocols</h2>
          <button class="tablinks" onclick="openTab(event, 'gene assembly')">Gene Assembly </button>
          <button class="tablinks" onclick="openTab(event, 'screen1')">Stage 1 Screening</button>
          <button class="tablinks" onclick="openTab(event, 'screen2')">Stage 2 Screening</button>
          <h2 class="tab-section">Generic protocols</h2>
          <button class="tablinks" onclick="openTab(event, 'Electrocompetent Cells')">Electrocompetent TOP10</button>
          <button class="tablinks" onclick="openTab(event, 'Electroporation')">Electroporation</button>
          <button class="tablinks" onclick="openTab(event, 'Gel Extraction')">Gel Extraction</button>
          <button class="tablinks" onclick="openTab(event, 'LB Plates')">LB Plates</button>
          <button class="tablinks" onclick="openTab(event, 'Plasmid Purification')">Plasmid Purification</button>
        </div>

        <div id="Gel Extraction" class="tabcontent">
          <h3>DESCRIPTION:</h3>
          <h3>MATERIALS AND APPARATUS:</h3>
          • Isopropanol (100%)<br>
          • Water bath at 50°C<br>
          • Buffer PE with 96-100% ethanol (see bottle label for volume)<br>
          • Centrifuge at 13,000 rpm<br>
          • Scalpel<br>
          • Buffer QG
          <h3>PROCEDURE:</h3>
          Steps:<br><br>
          1. Excise the DNA fragment from the agarose gel with a clean, sharp scalpel.<br><br>
          2. Weigh the gel slice in a colorless tube. Add 3 volumes Buffer QG to 1 volume gel
          (100 mg gel ~100 μl). <br><br>
          NOTE: The maximum amount of gel per spin column is 400 mg. For >2% agarose gels, add 6 volumes Buffer
          QG.<br><br>
          3. Incubate at 50°C for 10 min (or until the gel slice has completely dissolved). Vortex the
          tube every 2–3 min to help dissolve gel. After the gel slice has dissolved completely,
          check that the color of the mixture is yellow (similar to Buffer QG without dissolved
          agarose). <br><br>
          NOTE: If the color of the mixture is orange or violet, add 10 μl 3 M sodium acetate, pH 5.0, and mix. The
          mixture turns yellow.<br><br>
          4. Add 1 gel volume isopropanol to the sample and mix.<br><br>
          5. Place a QIAquick spin column in a provided 2 ml collection tube. Apply the sample to the QIAquick column
          and
          centrifuge for 1 min. Discard flow-through and place the QIAquick column back into the same tube. <br><br>
          NOTE: For sample volumes >800 μl, load and spin/apply vacuum again.<br><br>
          6. If DNA will subsequently be used for sequencing / in vitro transcription / microinjection,
          add 500 μl Buffer QG to the QIAquick column and centrifuge for 1 min. Discard flow-through and place the
          QIAquick column back into the same tube.<br><br>
          7. To wash, add 750 μl Buffer PE to QIAquick column and centrifuge for 1 min. Discard flow-through and place
          the
          QIAquick column back into the same tube.<br><br>
          NOTE: If the DNA will be used for salt-sensitive applications (e.g., sequencing, bluntended ligation), let the
          column stand 2–5 min after addition of Buffer PE. Centrifuge the QIAquick column in the provided 2 ml
          collection
          tube for 1 min to remove residual wash buffer.<br><br>
          8. Place QIAquick column into a clean 1.5 ml microcentrifuge tube.<br><br>
          9. To elute DNA, add 50 μl Buffer EB to the center of the QIAquick membrane and centrifuge the column for 1
          min.
          Add 30 μl Buffer EB to the center of the QIAquick membrane again, let the column stand for up to 4 min, and
          then
          centrifuge for 1 min. <br><br>
          NOTE: If purified DNA is to be analyzed on a gel, add 1 volume of Loading Dye to 5 volumes of purified DNA.
          Mix
          the solution by pipetting up and down before loading the gel.<br><br>
          </p>
        </div>

        <div id="LB Plates" class="tabcontent">
          <h3>DESCRIPTION:</h3>
          <h3>MATERIALS AND APPARATUS:</h3>
          • 2.5g yeast extract<br>
          • 5g peptone <br>
          • 5g NaCl<br>
          • 7.5g agar-agar<br>
          • 1L sterile bottle (check to see if this fits in the microwave for reheating)<br>
          • Sterile dH2O<br>
          • Sterile Petri dishes (Hackspace should have ~100mm diameter, holds ~20mL agar, should be able to see 100-200
          colonies)<br>
          • Ice bucket for keeping antibiotic on<br>
          • Chloroamphenicol antibiotic stored at 1000X concentration, dissolved in EtOH <br>
          o Recommended stock conc = 25mg/mL, working concentration = 25ug/mL<br><br>
          <h3>PROCEDURE:</h3>
          Per 500mL molten LB agar: (25 plates)<br><br>
          NOTE: This makes 25 plates, but we only need 16 plates total for Stage 1. <br><br>
          1. Weigh out the following into a 1L sterile bottle (20g total, pre-mixed powder in Hackspace contains first
          three components, then add 7.5g agar-agar separately):<br>
          • 2.5g yeast extract<br>
          • 5g peptone <br>
          • 5g NaCl<br>
          • 7.5g agar-agar<br>
          2. Make up solution to 500mL using sterile dH2O, gently swirl to mix and minimise amount of powder left on
          sides
          of bottle.<br><br>
          3. Screw lid on bottle loosely and label with autoclave tape.<br><br>
          4. Autoclave at 121C for 20 minutes<br><br>
          5. Remove solution from autoclave and cool to 50C (just about hot to touch but not too hot).<br><br>
          NOTE: Don’t need to use water bath for cooling to specific temperature unless generating a lot of plates &
          need
          to stop the agar from solidifying.<br><br>
          6. If making LB-Chlor plates, add 500uL of 25mg/mL antibiotic to solution and swirl to mix.<br><br>
          7. Pour 20mL of molten LB agar per 10cm diameter Petri dish. This should be conducted aseptically in a fume
          hood. Use 25mL pipette with Pipetboy to measure exact volumes. <br><br>
          NOTE: Do not eject volume from pipette too quickly, or bubbles will be introduced. Bubbles can be removed
          using
          a small pipette tip. When pouring, remove lid to dish and remove lid to bottle, then pour as specified above.
          Work quickly, as the agar will harden very quickly so pouring should be done in 5 minutes.<br><br>
          8. Place the lids on the plates and allow them to cool for 45 minutes or until solidified, then invert plates.
          To avoid condensation on the plates, stack 4-5 plates together, then stack the stacks so that the only plates
          that will accumulate condensation on the lid are the ones at the top of the stack. Store sealed with Parafilm
          at
          4C. <br><br>
          NOTE: Dried plates should be taped shut and stacked upside down on their lids in a container to protect from
          light. Write the date of preparation on the plates, make sure plates are not more than 2 months old.<br><br>
          </p>
        </div>

        <div id="Electrocompetent Cells" class="tabcontent">
          <h3>DESCRIPTION:</h3>
          <h3>MATERIALS AND APPARATUS:</h3>
          • Sterile distilled water<br>
          • 10% glycerol<br>
          • Graduated cylinder <br>
          • Weighing Scale/Boats<br>
          • Microcentrifuge tubes (1.5mL eppys)<br>
          • Hackspace-provided Eppendorf 5810R Centrifuge<br>
          o Manual for use, see Section 5. <br>
          • Fume hood for aseptic technique<br>
          • 1L sterilised bottle<br>
          • Ice bucket<br>
          • 1mL cuvettes <br>
          • Spectrophotometer<br>
          • 4x 50mL Falcon tubes stored at –20C.<br>
          • Sterile toothpick or Gilson pipette tip<br>
          • Dry ice/liquid nitrogen bath<br><br>
          SOB (Super Optimised Broth)<br><br>
          NOTE: If available in dry premixed form in the Hackspace, check ratios of each component. <br><br>
          • 2% (w/v) tryptone or peptone<br>
          • 0.5% (w/v) yeast extract<br>
          • 10 mM NaCl<br>
          • 2.5 mM KCl<br>
          • 20 mM MgSO4<br><br>
          Per 500mL: <br><br>
          1. Dissolve the following in 400mL of distilled water (ddH2O) in a 1L bottle:<br>
          • 10g tryptone or peptone<br>
          • 2.5g yeast extract <br>
          • 0.292g NaCl<br>
          • 0.093g KCl<br>
          • 1.2g MgSO4 <br><br>
          2. Bring final volume to 500mL with more distilled water (ddH2O). Adjust to pH 7.5 before use, requires 1M
          NaOH
          solution, pipette 1mL at a time.<br><br>
          10% (v/v) glycerol<br><br>
          • Dilute 1 volume of molecular biology-grade glycerol in 9 volumes of sterile pure H2O. <br>
          • Sterilise the solution by passing it through a pre-rinsed 0.22um filter. <br>
          • Store in 200mL aliquots at 4C. <br><br>
          <h3>PROCEDURE:</h3>
          NOTE: All steps should be carried out aseptically in the fume hood, unless otherwise stated, keep all reagents
          on ice, time ~ 55 min total.<br><br>
          Day 1:<br><br>
          1. Streak out stock of previously-competent TOP10 cells onto an LB plate with no antibiotics to obtain single
          colonies (if LB plate needs to be made, see this protocol for setting plates). Cover with lid and incubate
          overnight at 37C. <br><br>
          2. Autoclave:<br>
          2.1. 500mL SOB medium (not going to use it all)<br>
          2.2. 10% glycerol <br>
          2.3. Microcentrifuge tubes<br>
          2.4. 50mL Falcons (for centrifuging)<br><br>
          Day 2:<br><br>
          1. Pre-warm: Pre-warm 50mL SOB medium at 37C in a water bath at the start of day 2. <br><br>
          2. Reserve control: Keep 10mL SOB medium WITHOUT bacteria in a small bottle or Falcon to act as blank sample
          when measuring OD600 on Day 3. <br><br>
          3. Starter culture: At end of day, prepare 50mL sterile Falcon containing 10mL SOB medium. Using a sterile
          toothpick or pipette tip, pick up a single colony of cells from incubated Day 1 plate, then drop into Falcon.
          Cover flask with foil and incubate this starter culture overnight at 37C in a rotary shaker set to 225rpm.
          Prewarm the SOB medium in the same incubator, to be used the following day. <br><br>
          NOTE: To grow, bacteria flasks should never be filled with more than a quarter of its max volume, otherwise
          the
          aeration will be limiting. <br><br>
          To prep for Day 3:<br><br>
          Chill the following in 4C fridge: <br>
          • Centrifuge rotor (ask Hackspace)<br>
          • 1L 10% glycerol<br><br>
          Day 3:<br><br>
          1. In a flask, inoculate 2.5mL of the starter culture in 250mL fresh SOB medium (the volumes of starter
          culture
          and SOB should always be at a 1:100 ratio). <br><br>
          2. Incubate at 37C in a rotary shaker set to 225rpm until the OD600 is between 0.4-0.6 (0.4 most
          common).<br><br>
          NOTE: Optimal transformation efficiencies are achieved when E. coli cells are at an optical density, OD600nm
          ~0.4-0.5. DO NOT LET IT GO ABOVE 0.5. To obtain this biomass, the cell suspension should be checked every 30
          minutes using a spectrophotometer. It usually takes between 2-3 hours after inoculation of the culture to
          reach
          this point. There is a 10 minute window to transfer the flask to ice (see next step).<br><br>
          3. Once target OD600 is hit, transfer flask to ice, then transfer all contents of the flask to a 4x fresh 50mL
          Falcon tubes and keep on ice for 5-10 min. Swirl occasionally to ensure that cooling occurs evenly. <br><br>
          NOTE: After this point, do not let cells warm up past 4C. When pouring off supernatant, be careful not to
          disturb the pellet. If the pellet seems loose, re-spin the cells down. Aspirate with a pipette if necessary.
          <br><br>
          4. Centrifuge 1: Pellet the cells by centrifuging at 4000rpm for 5 minutes at 4C. <br><br>
          5. Pour off supernatant. Aspirate any residual broth. Resuspend each Falcon tube pellet in 10mL ice-cold
          sterile
          distilled water, then add another 40mL (so total = 50mL water added per Falcon). <br><br>
          6. Centrifuge 2: Pellet the cells again by centrifuging at 4000rpm for 5 minutes at 4C.<br><br>
          7. Resuspend in 2x 50mL distilled water (start with 4 tubes, resuspend each pellet in 10mL water, then
          transfer
          the contents of Tube 1 to Tube 2 and the contents of Tube 3 to Tube 4, then add 30mL distilled water to each
          of
          the remaining 2 tubes). <br><br>
          NOTE: At the end of this process, you should have 2 empty Falcons, and 2 full Falcons. <br><br>
          8. Centrifuge 3: Pellet the cells again by centrifuging at 4000rpm for 5 minutes at 4C.<br><br>
          9. Resuspend in 2x 50mL 10% glycerol. <br><br>
          10. Centrifuge 4: Pellet the cells again by centrifuging at 4000rpm for 7 minutes at 4C. <br><br>
          11. Pour away supernatant slowly, then resuspend the cells in a final volume of 200uL of ice-cold 10%
          glycerol.<br><br>
          NOTE: At this stage, you should have a total volume of 400uL cell suspension, divided into 2x 200uL in
          Falcons.
          At this stage, either move onto electroporation protocol or store away. <br><br>
          12. Snap-freeze: Pre-cool 10x 1.5mL eppys on dry ice, keep lids open. Aliquot 40uL cells into each tube, work
          quickly. Store in –80C freezer.<br><br>
          </p>
        </div>

        <div id="Electroporation" class="tabcontent">
          <h3>DESCRIPTION:</h3>
          <h3>MATERIALS AND APPARATUS:</h3>
          • Electrocompetent TOP10 cells stored in –80C freezer<br>
          • Plasmid to be transformed into cells<br>
          • Ice bucket<br>
          • LB-Chlor plates for selection <br>
          • Graduated cylinder <br>
          • Weighing Scale/Boats (check mg scale works)<br>
          • Microcentrifuge tubes (1.5mL eppys) & 0.1cm electroporation cuvettes <br>
          o Quantity: Each sample to be electroporated needs one fresh 1.5mL microcentrifuge tube and one 0.1cm cuvette.
          <br>
          • Hackspace-provided Eppendorf 5810R Centrifuge<br>
          • LB-Chlor liquid broth<br>
          • 10% (v/v) glycerol: 12.6g glycerol in 90mL of distilled water. Autoclave or filter sterilise. <br>
          • SOC recovery medium<br><br>

          LB broth/agar:<br><br>
          Per 500mL LB agar: (25 plates)<br><br>
          • 2.5g yeast extract<br>
          • 5g peptone <br>
          • 5g NaCl<br>
          • 7.5g agar-agar<br>
          • 1L sterile bottle (check to see if this fits in the microwave for reheating)<br>
          • Ice bucket for keeping antibiotic on<br>
          • Chloroamphenicol antibiotic stored at 1000X concentration, dissolved in EtOH <br>
          o Recommended stock conc = 25mg/mL, working concentration = 25ug/mL<br><br>
          Procedure: per 500mL molten LB agar = 25 plates<br><br>
          NOTE: This makes 25 plates, but we only need 16 plates total for Stage 1. <br><br>
          1. Weigh out the following into a 1L sterile bottle (20g total, pre-mixed powder in Hackspace contains first
          three components):<br>
          • 2.5g yeast extract<br>
          • 5g peptone <br>
          • 5g NaCl<br><br>
          If making LB agar plates, add:<br>
          • 7.5g agar-agar<br><br>
          2. Make up solution to 500mL using sterile dH2O, gently swirl to mix and minimise amount of powder left on
          sides
          of bottle.<br><br>
          3. Screw lid on bottle loosely and label with autoclave tape<br><br>
          4. Autoclave at 121C for 20 minutes<br><br>
          5. Remove solution from autoclave and cool to <50C (just about hot to touch but not too hot).<br><br>
            NOTE: Don’t need to use water bath for cooling to specific temperature unless generating a lot of plates &
            need to stop the agar from solidifying.<br><br>
            6. If making LB-Chlor plates, cool the LB medium down further until near room temperature and add 500uL of
            antibiotic to solution and swirl to mix.<br><br>

            SOC (Super Optimised Broth with Catabolite Repression)<br><br>
            Per litre of SOB:<br><br>
            NOTE: If available in dry premixed form in the Hackspace, check ratios of each component. <br><br>
            • 2% (w/v) tryptone/peptone<br>
            • 0.5% (w/v) yeast extract<br>
            • 10 mM NaCl<br>
            • 2.5 mM KCl<br>
            • 20 mM MgSO4<br><br>
            1. Dissolve the following in 500mL of distilled water (ddH2O): <br>
            • 10g tryptone/peptone <br>
            • 5g yeast extract <br>
            • 0.292g NaCl<br>
            • 0.093g KCl<br>
            • 1.2g MgSO4 <br><br>
            2. Bring final volume to 500mL with more distilled water (ddH2O)<br><br>
            Adjust to pH 7.5 before use, requires approx. 25mL of 1M NaOH solution per litre of SOB made (so instead of
            using 200mL water to bring final volume to 1L, use 175mL water + 25mL 1M NaOH.)<br><br>
            3. Cool SOB medium to <50C, then add 10ml syringe filter-sterilised 20% (w/v) glucose solution<br><br>
              NOTE: To filter-sterilise the glucose solution, attach a single-use, sterile 0.2um pore filter to a
              syringe
              tip, and screw down the syringe to secure it. Position the syringe at the mouth of a sterile Falcon tube,
              then pour the glucose solution into the syringe and gently push down the plunger. <br><br>

              Preparation steps for previous day:<br>
              • Pre-warm 2mL SOC recovery medium and LB-Chlor agar plates in 37C incubator (no shaking necessary) for at
              least 1 hour. <br>
              • Make 50% glycerol<br><br>
              NOTE: For each 40uL aliquot of electrocompetent cells, use 2mL SOC. Use this ratio to scale up for each
              transformation reaction you want to do. # cells in aliquot doesn’t matter, colonies will be grown out of
              them anyway. <br><br>
              • Pre-chill in 4C freezer: plasmid DNA, electroporation cuvettes, microcentrifuge tubes, distilled water,
              and 10% glycerol. <br><br>

              <h3>PROCEDURE:</h3>>
              1. Thaw the 40uL aliquots of electrocompetent cells on ice. Add 2 µL of purified DNA from Miniprep.
              <br><br>
              2. Mix by gently flicking the tube containing the electrocompetent cell + DNA mixture. Do not handle cells
              vigorously as this will reduce transformation efficiency. Let the mixture sit on ice for 1 minute.<br><br>
              3. Pipette the mixture into a chilled 0.1 cm electroporation cuvette, gently tap the cuvette on a flat
              surface to ensure the mixture is at the bottom of the cuvette.<br><br>
              4. Wipe any condensation off the sides of the cuvette before placing the cuvette in the chamber
              slide.<br><br>
              5. Push the slide into the chamber until the cuvette is seated between the contacts in the base of the
              chamber. Ensure the pulser is set to “Ec1” and press the "Pulse" button once.<br><br>
              6. Remove the cuvette from the chamber and immediately add 1mL of pre-warmed SOC recovery medium to the
              cuvette. Quickly and gently resuspend cells with a pipette. <br><br>
              NOTE: you need to be quick about transferring the cells to SOC medium; delaying the transfer between the
              stage at which you apply the pulse to the stage where SOC medium is added is crucial for recovering
              transformants. Delaying by a single minute causes a 3-fold drop in transformation. <br><br>
              7. Transfer the mixture to a new 1.5 mL microcentrifuge tube.<br><br>
              8. Incubate for 60 minutes at 37°C in a shaking incubator at 225rpm – ensure the tube lays flat.<br><br>
              9. Prepare two plates:<br>
              a. Streak plate 50uL of the cell mixture on a LB-Chlor plate. Incubate overnight at 37C. <br>
              b. Spin the remainder of cells at 3000 rpm for 2 minutes. Remove all but 50 μL media (otherwise you will
              lose pellet). Streak plate with remaining cells on a LB-Chlor plate. Incubate overnight at 37°C.<br><br>
              Making glycerol stocks of transformed cells: <br><br>
              1. Prepare autoclaved liquid LB-Chlor medium. Reserve 10mL of LB-Chlor for blank when measuring
              OD600.<br><br>
              2. Making an overnight culture: pick a colony of transformed cells from the LB-Chlor plate and add to 5mL
              LB-Chlor in 50mL Falcon (for mini-prep of plasmid). Incubate the tube overnight at 37C at 225rpm.<br><br>
              3. Next morning, take 2 ml for miniprep, use remaining for glycerol stock.<br><br>
              4. Add 1 mL of the overnight culture to an equal amount of 50% filtered glycerol in a 2 mL screw top tube
              or
              Eppys and gently mix by pipetting 5-6 times. <br><br>
              Note: Adding the 50% glycerol to an equal volume of cells effectively dilutes it down to 25%<br><br>
              5. Freeze the glycerol stock tube at -80°C. Subsequent freeze and thaw cycles reduce shelf life.<br><br>
              6. To recover bacteria from your glycerol stock, open the tube and use a sterile loop, toothpick or
              pipette
              tip to scrape some of the frozen bacteria off the top. Do not let the glycerol stock unthaw. Keep on ice
              block.<br><br>
              7. Grow bacteria overnight at 37C, for plasmid miniprep the following day. <br><br>
              Sources: Horizons prototyping + electroporation machine manual, Addgene glycerol stock page.<br><br>

              </p>
        </div>

        <div id="Plasmid Purification" class="tabcontent">
          <h3>DESCRIPTION:</h3>
          <p>
            Background: all QIAGEN plasmid purification protocols are based on a modified alkaline lysis procedure,
            followed
            by binding plasmid DNA to resin under low-salt and pH conditions. A medium-salt wash is then conducted to
            remove
            RNA, proteins, dyes, and other low-MW contaminants. The plasmid DNA is then eluted in a high-salt buffer and
            concentrated and desalted by isopropanol precipitation.<br><br>



            NOTE: Protocol is optimised for use with cultures grown to cell density of approximately 3-4 x 10^9 cells
            per
            mL. Ensure OD600 = 0.5.<br><br>



          <h3>MATERIALS AND APPARATUS:</h3>>

          • Ice<br>

          • Isopropanol<br>

          • Ethanol, 70%<br>

          • Plasmid resuspension buffer (e.g., TE buffer, pH 8.0, or Tris·Cl, pH 8.5 - check if we have any...)<br>

          • Tabletop microcentrifuge<br>

          • 1.5mL Eppy tubes<br><br>



          <h3>MATERIALS IN THE MINIKIT (12125):</h3>

          • x100 QIAGEN-tip 20<br>

          • 2 x 20 ml Buffer P1<br>

          • 2 x 20 ml Buffer P2<br>

          • 2 x 20 ml Buffer P3<br>

          • 2 x 60 ml Buffer QBT<br>

          • 2 x 240 ml Buffer QC<br>

          • 140 ml Buffer QF<br>

          • 2 x 2 mg RNase A (check if 10 mg/ml or 100 mg/ml solution)<br>

          • 2 x 20 ul LyseBlue<br><br>

          Prep steps:<br><br>

          1. Centrifuge RNase A briefly, then add the whole bottle into Buffer P1 (which gives a final concentration of
          100
          ug/mL). Then add the whole bottle of LyseBlue into Buffer P1.<br><br>

          2. Check Buffer P2 for SDS precipitation<br><br>

          3. Prechill (one bottle of) Buffer P3 at 4°C<br><br>

          4. Make ice<br><br>

          5. Chill centrifuge<br><br>





          <h3>PROCEDURE:</h3>

          1. Grow cells to reach a cell density of approximately 3–4 x 109 cells per milliliter. (For 1 ml of cells, OD
          =
          5;
          for 5 ml of cells, OD =~1)<br><br>

          - NOTE: Use a tube or flask with a volume of at least 4 times the volume of the culture for these:<br><br>

          - Pick a single colony to inoculate in a 3mL LB medium starter culture containing 25ug/uL chloramphenicol.
          Incubate for 8h at 37°C in a rotary shaker at 300 rpm.<br><br>

          2. Centrifuge cells at 6000 x g for 15 min at 4°C.<br><br>

          NOTE: If you wish to stop the protocol and continue later, freeze the cell pellets at −30 to −15°C.<br><br>

          3. Resuspend the bacterial pellet in 0.3 ml of Buffer P1 (ensure RNase A has been already added) in a 2 ml
          Eppy
          (for centrifuging).<br><br>

          4. Add 0.3 ml of Buffer P2, mix thoroughly by vigorously inverting the sealed tube 4−6 times, and incubate at
          room
          temperature for 5 min.<br><br>

          NOTE: Do not vortex. Do not leave solution for longer than 5mins. Close buffer P2 quickly (to avoid CO2
          reacting
          with solution).<br><br>

          5. Add 0.3 ml of (pre-chilled) Buffer P3, mix immediately and thoroughly by vigorously inverting 4–6 times,
          and
          incubate on ice for 5 min.<br><br>

          NOTE: After addition of Buffer P3, a fluffy white material forms (contains proteins/DNA etc) and the lysate
          becomes less viscous. If the mixture still appears viscous, more mixing is required to completely neutralize
          the
          solution.<br><br>

          6. Mix the sample (by inverting). Centrifuge at maximum speed in a microcentrifuge for 10 min.<br><br>

          7. (while centrifuge runs) set up a QIAGEN-tip 20 on a tube rack with a waste beaker under the column to
          collect
          the flow-through. Add 1ml of Buffer QBT (equilibration buffer) into the QIAGEN-tip 20, allow it to flow
          through
          the column completely before pouring in the supernatant in the next step.<br><br>

          NOTE: Try not to let the column dry out<br><br>

          8. (once centrifuge is finished) keep the supernatant containing plasmid DNA – pour it into the column
          quickly.
          The
          supernatant (containing plasmid DNA) will be clear, the precipitate (containing genomic GNA) will be
          white.<br><br>

          NOTE: After centrifugation, the supernatant should be clear - centrifuge again if needed.<br><br>

          9. Add 2 ml Buffer QC into the QIAGEN-tip 20 and let flow through. Then add again 2 ml Buffer QC into the
          QIAGEN-tip 20 and let flow through.<br><br>

          10. Remove waste beaker from under the column and replace with 1.5 ml Eppy. Add 0.8 ml Buffer QF to elute the
          plasmid DNA into the Eppy.<br><br>

          11. Take 2 µL of the elute and use Nanodrop machine (ds DNA setting) to quantify the amount (ideally around 50
          ng
          per µL of DNA), and get a purity score and quality score (ideally above 1.8-2.1). Blank first with 2 µL of
          elution buffer.<br><br>



          Complete following steps if scores are below 1.8-2.1 or if concentration is below 10 ng per µL. (Because the
          Nanodrop cannot accurately read concentrations below 10 ng per µL).<br><br>

          12. Precipitate DNA by adding 0.7 volumes (e.g. 0.56 ml per 0.8 ml of elution volume) of room-temperature
          isopropanol to the eluted DNA. Mix and centrifuge immediately at ≥15,000 x g rpm for 30 min in a
          microcentrifuge. Remove the supernatant using a pipette (without touching the pellet).<br><br>

          NOTES: All solutions should be at room temperature to minimize salt precipitation. Isopropanol pellets have a
          glassy appearance and may be more difficult to see than the fluffy, salt-containing pellets that result from
          ethanol precipitation. Marking the outside of the tube before centrifugation allows the pellet to be easily
          located. Isopropanol pellets are more loosely attached to the side of the tube; take care when removing the
          supernatant.<br><br>

          13. Wash DNA pellet with 1 ml of 70% ethanol and centrifuge at 15,000 x g for 10 min.
          Carefully decant the supernatant without disturbing the pellet.<br><br>

          14. Open the tube, gently blot off excess ethanol, and leave to air-dry the pellet for 5–10 min, and
          redissolve
          the
          DNA in a suitable volume (0.2 - 0.5 ml) of elution buffer.<br><br>

          15. Re-measure DNA quantity using Nanodrop (purity/quality scores should be ideally above 1.8-2.1), and
          concentration should be around 50 ng per µL.<br><br>



          2 µL of the purified DNA will be used for each round of electroporation. Any remaining DNA can be stored at
          room
          temperature.<br><br>



          Troubleshooting: take samples at step 7, 9, 10 and 11 to run agarose gel analysis and see where the problem
          occurred.<br><br>

          <a
            href="https://www.qiagen.com/gb/resources/download.aspx?id=0bd0c5fb-c271-43e7-af43-32d539374fa9&lang=en">Source</a><br><br>
          </p>
        </div>
      </div>

    </div>
    </div>
      



  </div>
</body>

<a href="#">
  <img id="back-to-top" src="../../content/back-to-top.svg">
</a>

<footer>Imperial College London iDEC 2021 Team ©</footer>
</html>